linter/lints/cpcps/lint_crl_has_idp.go

```diff
@@ -69,91 +88,100 @@ func (l *crlHasIDP) Execute(c *x509.RevocationList) *lint.LintResult {
 		}
 	}
 
```

Comment:
Update the comment on line 82 to either mention all three fields we care about, or none of them.